src/cacheExchange.ts

```diff
@@ -269,6 +269,8 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
   return ops$ => {
     const sharedOps$ = pipe(ops$, share);
 
+    // Buffer operations while waiting on hydration to finish
+    // If no hydration takes place we replace this stream with an empty one
```

Comment:
This is amazing! For Apollo requirement was to always hold of with the page rendering until store is rehydrated. 
Thanks to buffering rendering are not blocked and developers can see some of the wireframes of the app